plugins/inputs/ceph/ceph.go

```diff
@@ -48,6 +35,19 @@ type Ceph struct {
 	schemaMaps map[socket]perfSchemaMap
 }
 
+const (
+	measurement = "ceph"
+	typeMon     = "monitor"
+	typeOsd     = "osd"
+	typeMds     = "mds"
+	typeRgw     = "rgw"
+	osdPrefix   = "ceph-osd"
+	monPrefix   = "ceph-mon"
+	mdsPrefix   = "ceph-mds"
+	rgwPrefix   = "ceph-client"
+	sockSuffix  = "asok"
+)
+
```

Comment:
I think having it at the top is OK, isn't it?

Reply:
Yes, my bad.